model_analyzer/config/input/config_defaults.py

```diff
@@ -12,11 +12,13 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
+
 #
 # Common defaults
 #
 
-DEFAULT_CHECKPOINT_DIRECTORY = './checkpoints'
+DEFAULT_CHECKPOINT_DIRECTORY = os.path.join(os.getcwd(), 'checkpoints')
```

Comment:
Does this mean that relative directories no longer work?

Reply:
Actually some of the qa tests test this (L0_export_metrics)

Reply:
I added some code to make sure.